accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraphConfiguration.java

```diff
@@ -105,6 +119,37 @@ private URI getHdfsRootDir() throws URISyntaxException {
             Map.Entry entrySet = (Map.Entry) entrySetObject;
             configuration.set("" + entrySet.getKey(), "" + entrySet.getValue());
         }
+
+        String hadoopConfDir = getString(HADOOP_CONF_DIR, null);
+        if (hadoopConfDir != null) {
+            LOGGER.info("hadoop conf dir", hadoopConfDir);
+            File dir = new File(hadoopConfDir);
+            if (dir.isDirectory()) {
+                for (String xmlFilename : HADOOP_CONF_FILENAMES) {
+                    File file = new File(dir, xmlFilename);
+                    if (file.isFile()) {
+                        LOGGER.info("adding resource: %s to Hadoop configuration", file);
+                        try {
+                            ByteArrayInputStream in = new ByteArrayInputStream(FileUtils.readFileToByteArray(file));
+                            configuration.addResource(in);
```

Comment:
Can this just be a FileInputStream or does it need to be converted to bytes first?